KifuSF/KFCPendingDonations.swift

```diff
@@ -63,14 +63,14 @@ extension KFCPendingDonations: KFPPendingDonationCellDelegate {
         let selectedDonation = self.donations[indexPath.row]
         sender.descriptorView.cancelStickyButton.contentView.isEnabled = false
         
-        RequestService.cancelRequest(for: selectedDonation) { (isSuccessfull) in
+        RequestService.cancelRequest(for: selectedDonation) { [unowned self] (isSuccessfull) in
```

Comment:
@Pondorasti you can also use `[weak self]`. But, here, there are no retain cycle, or memory link in this case